src/screens/ContactScreen/index.js

```diff
@@ -372,7 +436,6 @@ const ContactScreen = ({navigation}) => {
         onChangeText={onSearchTextChange}
         onClearText={() => onSearchTextChange('')}
         isLoading={isLoadingSearchResult}
-        // onPress={handleSearch}
       />
 
       {!isAnon && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onClearText` prop is added to the Search component which clears the search text when the clear button is clicked. This is a good UX improvement.



<!-- This is an auto-generated comment by OSS CodeRabbit -->